catalog/nsili/catalog-nsili-endpoint/src/test/java/com/connexta/alliance/nsili/endpoint/TestNsiliWebEndpoint.java

```diff
@@ -16,14 +16,17 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
 
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
 import javax.ws.rs.core.Response;
 
-public class TestNsiliWebEndpoint {
+import ddf.security.service.SecurityServiceException;
+
+public class TestNsiliWebEndpoint extends NsiliCommonTest {
```

Comment:
`TestNsiliCommon` might be a better name, the convention is generally to have `Test` as the first word.